app/helpers/pages_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module PagesHelper
+  def organize_into_categories(pages)
+    categories = {}
+    pages.each do |page|
```

Comment:
I didn't actually look for this, but I think there's a `group_by` method that might work.